inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/pojo/sort/SortClusterConfigResponse.java

```diff
@@ -31,7 +31,7 @@
     String msg;
     int code;
     String md5;
-    List<SortTaskConfig> tasks;
+    SortClusterConfig data;
```

Comment:
pojo class can move to inlong-common module.
And remove pojo class of sort-standalone module "org.apache.inlong.sort.standalone.config.pojo".
Use same class in sort-standalone and manager.